doc/classes/ItemList.xml

```diff
@@ -239,6 +242,11 @@
 			<argument index="1" name="custom_bg_color" type="Color">
 			</argument>
 			<description>
+				Sets the background color of [code]idx[/code] item to the specified [Color].
+				[codeblock]
+					var someString = "Some text"
+					someString.set_item_custom_bg_color(0,Color(100,0,0,1) # this will set the background color of the first item of the control to red.
```

Comment:
Add spaces after commas. Start the comment with a capitalized letter.

Reply:
`Color` uses float values between 0.0 and 1.0, so this should be `Color(1, 0, 0, 1)` for red, not 100.

Reply:
Color(100, 0, 0, 1) worked normally without giving me any errors and the background was set to red.
Maybe this is worth opening an issue?